### PR TITLE
added protonvpn documentation

### PR DESCRIPTION
**Pull request by deanso**
This is documentation for custom protonvpn configuration. If not appropriate here just close it.

## Breaking change

```txt
<placeholder>
```

## Proposed change

```txt
<placeholder>
```

## Type of change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (which adds functionality to this container)
- [ ] Breaking change (fix/feature causing existing functionality to break)

## Additional information

- This PR fixes or closes issue: ```fixes #```
- This PR is related to issue: ```relates to #```
- Link to documentation updated (if done separately): ```https://...```

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.

If user exposed functionality or configuration variables are added/changed:

- [ x] Documentation added/updated

### CODE REVIEW

File: docs/provider-specific.md [hunk 1]
```diff
@@ -118,3 +118,74 @@ services:
           max-size: 10m
       image: haugene/transmission-openvpn:latest
 ```
+
+### PROTONVPN
+
+[PROTONVPN](https://protonvpn.com/support/linux-openvpn/#preparation) provides `.ovpn` configuration files. Just download the one you want to connect with and which allows p2p.
```
Comment (ilike2burnthing) [inline docs/provider-specific.md:124]:
```suggestion
[PROTONVPN](https://protonvpn.com/support/linux-openvpn/#preparation) provides `.ovpn` configuration files. Just download the one you want to connect with and which allows P2P.
```

File: docs/provider-specific.md [hunk 2]
```diff
@@ -118,3 +118,74 @@ services:
...
+
+### Prerequisites:
+User needs to have a payed account.
```
Comment (ilike2burnthing) [inline docs/provider-specific.md:127]:
```suggestion
User needs to have a paid account.
```

File: docs/provider-specific.md [hunk 3]
```diff
@@ -118,3 +118,74 @@ services:
...
+
+1. download your protonvpn ovpn file from a destination which allows p2p.
```
Comment (ilike2burnthing) [inline docs/provider-specific.md:129]:
```suggestion
1. download your ProtonVPN ovpn file from a destination which allows P2P.
```

File: docs/provider-specific.md [hunk 4]
```diff
@@ -118,3 +118,74 @@ services:
...
+2. in the directory with your docker-compose file, create a directory: `mkdir protonvpn`
+3. copy your ovpn file (node-<country of choice>.protonvpn.net.udp.ovpn) from step 1 to the protonvpn directory
+4. add the environment vars below and add +pmp to your username if you want to use port forwarding.
+5. add the [update-port.sh](https://github.com/haugene/vpn-configs-contrib/blob/main/openvpn/protonvpn/update-port.sh) script for protonvpn from vpn-configs-contrib to the protonvpn directory of step 2.
```
Comment (ilike2burnthing) [inline docs/provider-specific.md:133]:
```suggestion
5. add the [update-port.sh](https://github.com/haugene/vpn-configs-contrib/blob/main/openvpn/protonvpn/update-port.sh) script for protonvpn from vpn-configs-contrib to the ProtonVPN directory of step 2.
```

File: docs/provider-specific.md [hunk 5]
```diff
@@ -118,3 +118,74 @@ services:
...
+            options:
+                max-size: 10m
+        ports:
+            - 9091:9091
+        image: haugene/transmission-openvpn
+
+```
+
+
+After starting your conntainer, the `peer listening port` in transmission should be open after a minute or so.
```
Comment (ilike2burnthing) [inline docs/provider-specific.md:167]:
```suggestion
After starting your container, the `peer listening port` in Transmission should be open after a minute or so. 
```

File: docs/provider-specific.md [hunk 6]
```diff
@@ -118,3 +118,74 @@ services:
...
+
+```
+
+
+After starting your conntainer, the `peer listening port` in transmission should be open after a minute or so. 
+
+If not you can jump in the container and run the script manually and see which error you get, or set the debug env variable: `- DEBUG=true` and look in the logging of your container for the output of the script `update-port.sh`
+
+
+To check which ip adress your vpn is currently connected to, run this script:
```
Comment (ilike2burnthing) [inline docs/provider-specific.md:172]:
```suggestion
To check which IP address your VPN is currently connected to, run this script:
```

File: docs/provider-specific.md [hunk 7]
```diff
@@ -118,3 +118,74 @@ services:
...
+
+f_find_all()
+{
+curl --silent ipinfo.io/$ext_ip
+}
+
+var_cont_name=$(f_container_name)
+ext_ip=$(docker exec $var_cont_name curl --silent "http://ipinfo.io/ip")
+echo "transmission vpn currently connected to IP adress: $ext_ip"
+echo "This ip adress is in the following country: "
```
Comment (ilike2burnthing) [inline docs/provider-specific.md:189]:
```suggestion
echo "Transmission VPN currently connected to IP address: $ext_ip"
echo "This IP address is in the following country: "
```

**Review by ilike2burnthing — CHANGES_REQUESTED** [review]

File: docs/provider-specific.md (hunk 4 above)
Comment (deanso) [reply docs/provider-specific.md:133]: The directory is named protonvpn in step2. So it wouldn't be logical to call the same directory in step 5 ProtonVPN?

Comment (ilike2burnthing) [reply docs/provider-specific.md:133]: Whoops, my bad, fixed the wrong word. Edited above.

**Pull request merged**